### PR TITLE
way-displays 1.5.3 -> 1.6.0

### PR DESCRIPTION
**Pull request by alex-courtis**
###### Description of changes

Attn @simoneruffini 

Refactor for 1.6.0 changed a source file, breaking postPatch.

Removed postPatch and added `ROOT_ETC` as discussed here: https://github.com/alex-courtis/way-displays/issues/45

I would be most grateful for your review / testing / advice.

I do not know how to encode the sha256. My attempts to encode the git hash for 1.5.3 gave different results...

[CHANGELOG.txt](https://github.com/alex-courtis/way-displays/blob/master/CHANGELOG.txt)

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/wayland/way-displays/default.nix [hunk 1]
```diff
@@ -18,10 +18,6 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-5A0qZRpWw3Deo9cGqGULpQMoPCVh85cWE/wJ5XSbVJk=";
```
Comment (teto) [inline pkgs/tools/wayland/way-displays/default.nix:18]: you need to update the sha

**Comment by teto** [comment]
> I do not know how to encode the sha256.

change a letter from the hash and try building. 
Otherwise you can use nix-prefetch-git

**Comment by alex-courtis** [comment]
> > I do not know how to encode the sha256.
> 
> change a letter from the hash and try building. Otherwise you can use nix-prefetch-git

Thanks for getting back to me.

Unfortunately I'm just the application owner and don't actually run NixOS... I'm on Arch.

If someone could dig up the details as to what fetchFromGitHub does or run it for me, I'd be most grateful.

**Comment by simoneruffini** [comment]
I can update the sha256, but I don't know what is the procedure to add commits to this PR. Do I need to clone @alex-courtis repo and push a PR to his branch?

**Comment by alex-courtis** [comment]
> I can update the sha256, but I don't know what is the procedure to add commits to this PR. Do I need to clone @alex-courtis repo and push a PR to his branch?

You are now a collaborator on git@github.com:NixOS/nixpkgs.git

I'd be really grateful if you pushed the changes.

**Review by simoneruffini — APPROVED** [review]

**Review by simoneruffini — APPROVED** [review]

**Pull request merged**